addons/web/static/src/legacy/scss/colorpicker.scss

```diff
@@ -1,4 +1,31 @@
 // COLOR PICKER
+.colorpicker {
+    --o-colorpicker-bg-rgb: 255, 255, 255;
+    --o-colorpicker-text-rgb: 0, 0, 0;
+
+    .o_colorpicker {
+        &_bg {
```

Comment:
Let's not use that feature that makes the code unreadable and most importantly: ungreppable :wink: